contracts/spec_interfaces/ICompliance.sol

```diff
@@ -6,19 +6,19 @@ import "@openzeppelin/contracts/ownership/Ownable.sol";
 
 /// @title Elections contract interface
 interface ICompliance /* is Ownable */ { // TODO rename to IValidatorIdentification? or make compliance API more generic?
-	event ValidatorConformanceUpdate(address validator, string conformanceType);
+	event ValidatorComplianceUpdate(address validator, string complianceType);
 
 	/*
      * External methods
      */
 
     /// @dev Called by a validator as part of the automatic vote unready flow
     /// Used by the Election contract
-	function getValidatorCompliance(address addr) external view returns (string memory conformanceType);
+	function getValidatorCompliance(address addr) external view returns (string memory complianceType);
 
     /// @dev Called by a validator as part of the automatic vote unready flow
     /// Used by the Election contract
-	function setValidatorCompliance(address addr, string calldata conformanceType) external /* Owner only */ ;
+	function setValidatorCompliance(address addr, string calldata complianceType) external /* Owner only */ ;
```

Comment:
Can we not have more than one compliance type? @OdedWx

The abstraction here is that a validator can comply with some requirement. so why not allow them to satisfy more than one? I know this might complicate the interfaces a little bit and its... YAGNI. just want to make sure we knowingly disallow more than one compliance type - in the future this may be more difficult to change. but supporting more than one compliance type will require a new elections contract anyways... so maybe thats fine. it looks a little unclean semantically thought. just pointing out.

Reply:
No longer relevant as we plan to represent as boolean